map.json

```diff
@@ -0,0 +1,19 @@
+{
+  "builder": "builder",
+  "controller": "controller",
+  "dockerbuilder": "dockerbuilder",
+  "fluentd": "fluentd",
+  "monitor": "monitor",
+  "logger": "logger",
+  "minio": "minio",
+  "nsq": "nsqd",
+  "postgres": "database",
+  "redis": "redis",
+  "registry": "registry",
+  "registry-proxy": "registry-proxy",
```

Comment:
Ah good point -- not quite merged yet (https://github.com/deis/workflow/pull/734) but should be by end of this milestone.  Looks like just one last change technically not released (https://github.com/deis/registry-proxy/compare/v1.1.1...master) but I assume we're fine with dropping that?  If so, I'll remove this line.